examples/save-to-disk/main.go

```diff
@@ -64,6 +64,7 @@ func main() {
 	// Set a handler for when a new remote track starts, this handler saves buffers to disk as
 	// an ivf file, since we could have multiple video tracks we provide a counter.
 	// In your application this is where you would handle/process video
+	mediaRecorder := []media.Writer{nil, nil}
```

Comment:
Can you put a `sync.Mutex` around this. Just need to lock since ICE+OnTrack fire in threads